### PR TITLE
add Accurate

### PR DESCRIPTION
**Pull request by umezawatakeshi**
part of https://github.com/cybozu-go/neco/issues/1704

- add [Accurate](https://github.com/cybozu-go/accurate)

Signed-off-by: UMEZAWA Takeshi <takeshi-umezawa@cybozu.co.jp>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,6 +8,16 @@ JSONNET_LIBS_K8S_ALPHA_VERSION = 1.20
...
 
+.PHONY: update-accurate
+update-accurate:
+	$(call get-latest-gh,cybozu-go/accurate)
+	rm -rf /tmp/accurate
+	cd /tmp; git clone --depth 1 -b $(latest_gh) https://github.com/cybozu-go/accurate
+	rm -rf accurate/base/upstream/*
+	cp -a /tmp/accurate/config/* accurate/base/upstream
+	rm -rf /tmp/accurate
+	sed -i -E 's/newTag:.*$$/newTag: $(subst v,,$(latest_gh))/' accurate/base/kustomization.yaml
```
Comment (ymmt2005) [inline Makefile:19]: There's no need to replace the image tag because Accurate is owned by us.

Comment (ymmt2005) [reply Makefile:19]: Ah, the file is not from the upstream.

File: accurate/base/namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: system
```
Comment (ymmt2005) [inline accurate/base/namespace.yaml:4]:
```suggestion
  name: accurate
```

File: Makefile (hunk 1 above)
Comment (umezawatakeshi) [reply Makefile:19]: The upstream sets image tag by [kustomization.yaml at repo root](https://github.com/cybozu-go/accurate/blob/e70599c62a283a4c05dfbe83cf2658022a40e008/kustomization.yaml). This kustomization.yaml also generates configMap but its content should be modified in neco-apps. So we cannot use the kustomization.yaml and write `newTag` by ourselves.

File: accurate/base/namespace.yaml (hunk 2 above)
Comment (umezawatakeshi) [reply accurate/base/namespace.yaml:4]: If so, the Namespace won't be patched.

... I checked `kustomize build` result of original kustomization.yaml, the name of the Namespace becomes `accurate-system`, not `accurate`. ...!?

Comment (ymmt2005) [reply accurate/base/namespace.yaml:4]: This is my result.

```console
$ kustomize build . | head
apiVersion: v1
kind: Namespace
metadata:
  labels:
    app.kubernetes.io/name: accurate
  name: accurate
```

Comment (ymmt2005) [reply accurate/base/namespace.yaml:4]: I guess your `kustomize` is an old version.

```
$ kustomize version
{Version:kustomize/v4.1.3 GitCommit:0f614e92f72f1b938a9171b964d90b197ca8fb68 BuildDate:2021-05-20T20:52:40Z GoOs:linux GoArch:amd64}
```

Comment (umezawatakeshi) [reply accurate/base/namespace.yaml:4]: Oops, got it.

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**